src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <nb-layout>
-
+  <h1>Welcome to nebular-demo!</h1>
```

Comment:
Let's rather remove a test that requires this element. I don't see any reason to add the heading as it doesn't render anyway.